src/components/global/footer.tsx

```diff
@@ -1,9 +1,223 @@
 import { EXAMPLE_PATH } from "../../lib/constants";
+import styled from "@emotion/styled";
+import { GridContainer } from "../global/grid/gridContainer";
+import Pill from "../global/pill";
+import { breakpoints, colors } from "../../styles/variables";
+import chatIcon from "../../../public/svg/icon-chat.svg";
+import Image from "next/image";
+import Link from "next/link";
+import { getRemSize } from "../../styles/globalCss";
+import eoanPicture from "../../../public/eoan-picture.png";
+import {
+  StyledParagraphWrapper,
+  StyledParagraphText,
+  StyledPillWrapper,
+  StyledTextSpacer
+} from "../../components/home/approach";
+
+const StyledWrapper = styled(GridContainer)`
+  margin: 140px 0;
```

Comment:
On most pages there is much more space from the last element to the logo text, I suggest increasing this (take a look at at the other pages too to make sure)
```suggestion
  margin: 300px 0 260px 0;
```

Reply:
Yeah, that previous margin was too small, now it looks better.